tests/disaster-recovery/regional-dr/test_failover_and_relocate.py

```diff
@@ -121,6 +130,37 @@ def test_failover_and_relocate(
         logger.info(f"Waiting for {wait_time} minutes to run IOs")
         sleep(wait_time * 60)
 
+        # Set cluster_kubeconfig value for 'drpc_obj' object to fetch the details without switching the cluster context
+        acm_cluster_kubeconfig = os.path.join(
+            config.clusters[get_active_acm_index()].ENV_DATA["cluster_path"],
+            config.clusters[get_active_acm_index()].RUN.get("kubeconfig_location"),
+        )
+        drpc_obj.cluster_kubeconfig = acm_cluster_kubeconfig
+
+        # Get lastGroupSyncTime before failover
+        drpc_data = drpc_obj.get()
+        last_group_sync_time = drpc_data.get("status").get("lastGroupSyncTime")
+        logger.info(
+            f"The value of lastGroupSyncTime before failover is {last_group_sync_time}."
+        )
+
+        # Verify lastGroupSyncTime before failover
+        time_format = "%Y-%m-%dT%H:%M:%SZ"
+        last_group_sync_time = datetime.strptime(last_group_sync_time, time_format)
+        current_time = datetime.strptime(
+            datetime.utcnow().strftime(time_format), time_format
+        )
+        time_since_last_sync = (
+            current_time - last_group_sync_time
+        ).total_seconds() / 60
+        logger.info(
+            f"Before failover - Time in minutes since the last sync {time_since_last_sync}"
+        )
+        assert (
+            time_since_last_sync < scheduling_interval
```

Comment:
This condition is incorrect IMO.
E.g. if the value of `time_since_last_sync` is more than scheduling_interval by 2x or 3x, warning and critical level alerts would be raised, meaning the sync is likely to recover or pointing towards connectivity issues.
Ensuring there aren't connevtivity issues, we can still failover, and even relocate, but relocate should be avoided if this difference is more than 2x or even 3x. The relocate would just wait for final sync to complete, so timeout should be increased accordingly in relocate tests before failing them.

Reply:
Also, how are we handling it when primary goes down and then we failover?

Reply:
Upated to 2 x scheduling interval as discussed.
The value is taken before bringing down the primary cluster.

Reply:
Updated to 2.5 x scheduling interval